spec.html

```diff
@@ -36557,27 +36557,138 @@
+        </emu-alg>
+      </emu-annex>
+      <emu-annex id="sec-web-compat-block-static-semantics-early-errors">
+        <h1>Changes to Block Early Errors</h1>
+        <p>In Static Semantics: Early Errors of <emu-grammar>Block : `{` StatementList `}`</emu-grammar> (<emu-xref href="#sec-block-static-semantics-early-errors"></emu-xref>) the following replaces the first list item:</p>
+        <ul>
+          <li>
+            It is a Syntax Error if the LexicallyDeclaredNames of StatementList contains any duplicate entries, unless the source code matching this production is not strict mode code and the duplicate entries are only bound by FunctionDeclarations.
+          </li>
+        </ul>
+      </emu-annex>
+      <emu-annex id="sec-web-compat-block-runtime-semantics-evaluation">
+        <h1>Changes to Block Runtime Evaluation</h1>
+        <p>During runtime evaluation of <emu-grammar>Block : `{` StatementList `}`</emu-grammar> (<emu-xref href="#sec-block-runtime-semantics-evaluation"></emu-xref>) the following step is performed in place of step 2:</p>
+        <emu-alg>
+          1. Let _blockEnv_ be NewBlockEnvironment(_oldEnv_).
+        </emu-alg>
+        <p>During runtime evaluation of <emu-grammar>SwitchStatement : `switch` `(` Expression `)` CaseBlock</emu-grammar> (<emu-xref href="#sec-switch-statement-runtime-semantics-evaluation"></emu-xref>) the following step is performed in place of step 4:</p>
+        <emu-alg>
+          1. Let _blockEnv_ be NewBlockEnvironment(_oldEnv_).
+        </emu-alg>
+      </emu-annex>
+      <emu-annex id="sec-web-compat-blockdeclarationinstantiation">
+        <h1>Changes to BlockDeclarationInstantiation</h1>
+        <p>During BlockDeclarationInstantiation (<emu-xref href="#sec-blockdeclarationinstantiation"></emu-xref>) the following step is performed in place of step 2:</p>
+        <emu-alg>
+          1. Assert: _envRec_ is a block Environment Record.
+        </emu-alg>
+        <p>The following steps are performed in place of step 4:</p>
+        <emu-alg>
+          1. Let _functionNames_ be an empty List.
+          1. Let _functionsToInitialize_ be an empty List.
+          1. For each _d_ in _declarations_, in reverse list order do
+            1. For each element _dn_ of the BoundNames of _d_ do
+              1. If IsConstantDeclaration of _d_ is *true*, then
+                1. Perform ! _envRec_.CreateImmutableBinding(_dn_, *true*).
+              1. Else,
+                1. Perform ! _envRec_.CreateMutableBinding(_dn_, *false*).
+            1. If _d_ is a |GeneratorDeclaration| production or a |FunctionDeclaration| production, then
+              1. Let _fn_ be the sole element of the BoundNames of _d_.
+              1. If _fn_ is not an element of _functionNames_, then
+                1. NOTE If there are multiple |FunctionDeclaration|s for the same name, the last declaration is used. There are only multiple declarations if _code_ is not contained in strict mode code.
```

Comment:
This note might be clearer if it stated "If there are multiple FunctionDeclarations for the same name, the last declaration _is the only declaration returned by LexicallyScopedDeclarations_.